.gitignore

```diff
@@ -5,6 +5,7 @@ submodules/
 subprojects/lua/
 subprojects/libagg/
 subprojects/reproc/
+data/core/start.lua
```

Comment:
I understand this is no longer participating to the final build but I don't think that it's a good idea to ignore it.

Reply:
that's a left over of the previous version with the removed script, I'll force push to clean up this.